library/src/main/camera2/com/google/android/cameraview/Camera2.java

```diff
@@ -218,6 +230,20 @@ public void onImageAvailable(ImageReader reader) {
 
     private int mDisplayOrientation;
 
+    private boolean mStartVideoRecording = false;
+
```

Comment:
Yeah but I wanted to keep the original formatting to minimize the amount of changed lines